scenarios/kubernetes_verify.py

```diff
@@ -62,7 +68,7 @@ def main(branch, script, force):
     remote = 'bootstrap-upstream'
     uri = 'https://github.com/kubernetes/kubernetes.git'
     try:
-        check('git', 'remote', 'remove', remote)
+        check_output('git', 'remote', 'remove', remote)
```

Comment:
I like the intent of this, and I would prefer we do it in a way that makes it smarter about whether we call git remote remote in the first place.

Will you refactor this to call `git remote show` and use that output to determine whether to call `git remote remove`?